docs/Other SDL Features/Setting the Built-in Nav System Destination/index.md

```diff
@@ -1,17 +1,20 @@
-# Setting the Navigation Destination
-Setting a Navigation Destination allows you to send a GPS location, prompting the user to navigate to that location using their embedded navigation. When using the `SendLocation` RPC, you will not receive a callback about how the user interacted with this location, only if it was successfully sent to Core and received. It will be handled by Core from that point on using the embedded navigation system.
+# Setting the Built-In Navigation Destination
```

Comment:
With app services, you can also use this RPC to set navigation on the active nav app.

Reply:
Also, the folder and config should be renamed.

Reply:
* Renamed sub-menu in config
* Fixed note and added links to **App Services**